### ISSUE TITLE
Steam refuses to open no matter what.

### ISSUE THREAD

**Issue by Babylion122** [Botspot/pi-apps#2449]
### Confirmations

- [X] I searched for similar bug reports in https://github.com/Botspot/pi-apps/issues and found none were relevant.
- [X] I tried updating Pi-Apps and the issue is still not fixed. [How to update Pi-Apps?](https://pi-apps.io/wiki/getting-started/updating-pi-apps/)
- [X] IMPORTANT: I promise to respond to comments on this issue and write a good bug report https://www.softwaretestinghelp.com/how-to-write-good-bug-report/

### What happened?

App installs but fails when attempting to install the libc6-i386 library.

### Description

Currently if you attempt to install Steam using PiApps on PiOS 64-bit it will fail when attempting to install a library. I have done research on this and I suspect that its because the library its attempting to download got taken down and moved, as I saw many similar threads on forums relating to Ubuntu x86 and PopOS asking why Steam or other apps that use that library won't open/install.

### What are your system specs (run the following command in your terminal)?

```bash
OS: Debian GNU/Linux 11 (bullseye)
OS architecture: 64-bit
Last updated Pi-Apps on: 09/21/2023
Latest Pi-Apps version: 09/21/2023
Kernel: aarch64 6.1.21-v8+
Device model: Raspberry Pi 4 Model B Rev 1.5 
Cpu name: Cortex-A72
Ram size: 7.81 GB
Raspberry Pi OS image version: 2023-05-03
Language: en_US.UTF-8
```

### (Recommended) Error log? Terminal output? Debug messages?

```bash
Package Install opens, fails, closes, then this is the output.

You are missing the following 32-bit libraries, and Steam may not run:
libc.so.6
```

**Comment by Botspot** [comment]
Thanks for reaching out. Please check the box64 version by running this command: `box64 -v`
Then paste its output to this github issue.

**Comment by Botspot** [comment]
I just installed Steam on my 64-bit Bullseye image and steam runs fine here. So the issue you are having does not appear to be affecting everybody.

**Comment by stealmyhousekey** [comment]
I also ran into this same error last night, but seem to have resolved it by running the [install.sh script from the box86 repo.](https://github.com/ptitSeb/box86/blob/master/install_steam.sh)

However, I'm now running into a new issue where Steam will never prompt me to login after updating (essentially the same issue [here](https://github.com/ptitSeb/box86/issues/833#issuecomment-1639016735)).

I've tried purging and reinstalling box86/64 and Steam, but no joy.

[Here are my logs.](https://pastebin.com/qjGBR3jR)

**Comment by theofficialgman** [comment]
> ### (Recommended) Error log? Terminal output? Debug messages?
> ```shell
> Package Install opens, fails, closes, then this is the output.
> 
> You are missing the following 32-bit libraries, and Steam may not run:
> libc.so.6
> ```

closing as user error. reboot once before starting as pi-apps steam install instructed you to do. otherwise you may inadvertently launch steam directly and not through the `/usr/local/bin/steam` script (which supplies required environment variables and tuning arguments)
